spec/vcr/applications_controller/show/non_lease_up_application.yml

```diff
@@ -0,0 +1,584 @@
+        Housing","city":null,"certificateNumber":null,"appMemberID":null,"applicationID":"a0o0P00000Fv20MQAR","address":null},{"zipCode":null,"state":null,"shortformPreferenceID":"a0w0P00000N1bbCQAR","requiresProof":false,"recordTypeDevName":"DTHP","preferenceProof":null,"postLotteryValidation":null,"optOut":false,"naturalKey":null,"lwPreferenceProof":null,"listingPreferenceID":"a0l0P00001Lx8XPQAZ","individualPreference":null,"city":null,"certificateNumber":null,"appMemberID":null,"applicationID":"a0o0P00000Fv20MQAR","address":null},{"zipCode":null,"state":null,"shortformPreferenceID":"a0w0P00000N1bbDQAR","requiresProof":false,"recordTypeDevName":"NRHP","preferenceProof":"Telephone
+        bill","postLotteryValidation":"Unconfirmed","optOut":false,"naturalKey":null,"lwPreferenceProof":null,"listingPreferenceID":"a0l0P00001Lx8XUQAZ","individualPreference":null,"city":null,"certificateNumber":null,"appMemberID":"a0n0P00000BEG2sQAH","applicationID":"a0o0P00000Fv20MQAR","address":null},{"zipCode":null,"state":null,"shortformPreferenceID":"a0w0P00000N1bbEQAR","requiresProof":false,"recordTypeDevName":"L_W","preferenceProof":null,"postLotteryValidation":"Unconfirmed","optOut":true,"naturalKey":null,"lwPreferenceProof":null,"listingPreferenceID":"a0l0P00001Lx8XeQAJ","individualPreference":null,"city":null,"certificateNumber":null,"appMemberID":null,"applicationID":"a0o0P00000Fv20MQAR","address":null},{"zipCode":null,"state":null,"shortformPreferenceID":"a0w0U000001IJBEQA4","requiresProof":false,"recordTypeDevName":"AG","preferenceProof":null,"postLotteryValidation":null,"optOut":false,"naturalKey":null,"lwPreferenceProof":null,"listingPreferenceID":"a0l0U000003lWZqQAM","individualPreference":null,"city":null,"certificateNumber":null,"appMemberID":null,"applicationID":"a0o0P00000Fv20MQAR","address":null}],"referral":"Newspaper;MOHCD
+        Website","processingStatus":null,"primaryApplicant":{"zip":"94114-2535","yCoordinate":"4545683.891332","xCoordinate":"-13629293.1193475","workInSf":"true","whichComponentOfLocatorWasUsed":"eas","webAppID":"1","state":"CA","sexualOrientationOther":"Ziggy
+        Stardust","sexualOrientation":"Not listed","sexAtBirth":null,"relationship":null,"race":"Other/Multiracial","preferenceStatus":null,"preferenceAddressMatch":"Not
+        Matched","phoneType":"Home","phone":"2222222222","noPhone":false,"noEmail":false,"noAddress":false,"nickname":null,"middleName":null,"maritalStatus":null,"mailingZip":"94114-2535","mailingState":"CA","mailingCity":"SAN
+        FRANCISCO","mailingAddress":"4053 18TH ST","lastName":"Doe","languageOther":null,"language":null,"hiv":false,"hasSameAddressAsApplicant":null,"hasAltMailingAddress":false,"genderOther":"Dothraki","gender":"Not
```

Comment:
this one looks more like a fake app

Reply:
this is an auto-generated app for the Automated Test Listing